README.md

```diff
@@ -0,0 +1,25 @@
+# Hand Raiser robot for Zoom meetings
+
+When we have large meetings, sometimes the remote employees are unable to participate in the Q&A section at the end, even though they're able to view the main meeting over Zoom. This repo provides a way for a robot to join the Zoom meeting and raise a physical hand in the (meatspace) meeting room when someone on Zoom has a question.
+
+## Installation
+
+1. Clone this repo locally.
+2. Run `pip install -r requirements.txt` to install the dependencies.
+3. Edit `secrets.py` so it contains the robot's secret and URL. You can get these from anyone who worked on this project.
+4. Step 3 probably made the repo dirty. Run `git update-index --skip-worktree secrets.py` to make the repo clean again.
+
+## Running
+1. When you want Hand Raiser Bot to join a meeting, run `./main.py '<url-of-zoom-meeting>'` to start it. The URL likely contains a question mark, so we recommend enclosing the entire URL in single quotes so your terminal doesn't try pattern-matching on it.
+2. This will open a Chrome window and join the Zoom meeting as the user "Hand Raiser Bot".
+3. Whenever someone in the Zoom meeting selects the "Raise Hand" reaction, we'll move the servo on the robot. It will be raised whenever _at least 1_ person in the Zoom meeting has their hand raised, and lowered again when no one has their hand raised.
+4. If Hand Raiser Bot's servo has been raised for at least 30 seconds, we will wiggle it side to side, to try to call attention to ourselves.
+5. When the meeting is over (or when you want Hand Raiser Bot to leave), hit control-C in the terminal to shut everything down. This will also lower the servo even if someone in the Zoom meeting still has their hand raised.
+
+## Code Layout
+- `robot.py` is how to talk to Viam to move the hardware itself. This can raise and lower the servo, and wiggle it if it has been raised for long enough.
```

Comment:
```suggestion
- `robot.py` talks to `viam-server` to move the hardware itself. This script raises and lowers the servo, and wiggles it if it has been raised for a while.
```
[opt]

Reply:
I like this except the word "script" is misplaced: this file acts as a library to be used elsewhere, and main.py is the "real" script. 

I'm new to suggested changes within a code review: how do I tweak your suggested change without just making the change on my laptop and then committing/pushing it up here?

Reply:
It is possible to edit peoples' suggestions but I try to avoid doing that since it gets confusing. Instead, you can just make your own new suggestion in this thread, copy-paste the contents of my suggestion into it, and comment that then commit it. IDK if this is right either:
```suggestion
- `robot.py` talks to `viam-server` to move the hardware itself. The code in this file raises and lowers the servo, and wiggles it if it has been raised for a while.
```